frontend/lib/Logging/utils.dart

```diff
@@ -0,0 +1,31 @@
+
+enum ButtonState { enabled, disabled }
+
+enum FormType { signin, signup }
+
+bool checkPasswordRegex(String text) {
+  const String pattern = r'^(?=.*[0-9])(?=.*[a-z])(?=.*[A-Z]).{1,}$';
+  final RegExp regex = RegExp(pattern);
+  return executeRegex(regex, text);
+}
+
+bool checkEmailRegex(String text) {
+  const String pattern =
+      r'^(?=.{1,64}@)[A-Za-z0-9_-]+(.[A-Za-z0-9_-]+)*@[^-][A-Za-z0-9-]+(.[A-Za-z0-9-]+)*(.[A-Za-z]{2,})$';
+  final RegExp regex = RegExp(pattern);
+  return executeRegex(regex, text);
+}
```

Comment:
W sumie jak `executeRegex` jest używane tylko w dwóch miejscach to można po prostu zrobić tak przy return'ach
```dart
return regex.hasMatch(text);
```
Sam się tego uczę, żeby takich krótkich funkcji nie robić, więc myślę, że wtedy będzie można się pozbyć `executeRegex`

Reply:
oki, to zmieniam i merguje